gradle/dependencies.gradle

```diff
@@ -20,9 +20,9 @@ libraries.bytebuddyandroid = "net.bytebuddy:byte-buddy-android:${versions.bytebu
 
 libraries.errorprone = 'com.google.errorprone:error_prone_core:2.3.1'
 
-libraries.objenesis = 'org.objenesis:objenesis:2.6'
+libraries.objenesis = 'org.objenesis:objenesis:3.0.1'
```

Comment:
Version 3.0.1 requires Java 8. If this is an issue, we need to keep using the older version.

Reply:
I reverted this change.